coreos-installer

```diff
@@ -591,6 +591,18 @@ write_image_to_disk() {
     udevadm settle
 }
 
+############################################################
+#Make call home HTTP(s) call as the final step
+############################################################
+call_home() {
+    CALLHOME_URL=$(cat /tmp/callhome_url)
+    if [ ! -z "$CALLHOME_URL" ]
```

Comment:
For my understanding: isn't this equivalent to `if [ -n "$CALLHOME_URL" ]`?

Reply:
I copy-pasted this, this is my very first dracut patch and I am really not sure if this is being executed by GNU bash or something else. So keeping.